src/Utility/TextSplitter.cpp

```diff
@@ -0,0 +1,29 @@
+//
+// Created by dawid on 23.10.2020.
+//
+
+#include "TextSplitter.h"
+
+int TextSplitter::rangeWidth(long long int textLength) {
+    return ceil((double) textLength / (double) numOfThreads);
```

Comment:
C style casts are not a recommended way to cast expressions. See `std::static_cast`

Reply:
Okey, copy that :+1: